package.json

```diff
@@ -138,8 +138,8 @@
   "husky": {
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-commit": "npm run style && git add -u",
-      "pre-push": "npm run lint"
+      "pre-commit": "npm run style && npm run lint",
+      "pre-push": "npm run lint && npm run test"
```

Comment:
I think the `npm run lint` during pre-push is redundant when it is already executed in the pre-commit hook.